doc/classes/SpringArm3D.xml

```diff
@@ -32,7 +32,7 @@
 			<return type="float">
 			</return>
 			<description>
-				Returns the proportion between the current arm length (after checking for collisions) and the [member spring_length]. Ranges from 0 to 1.
+				Returns the current length of the spring arm.
```

Comment:
We prefer to use this more compact form, as per the [Docs writing guidelines](https://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html):

```suggestion
				Returns the spring arm's current length.
```